js/controller.js

```diff
@@ -1231,9 +1231,6 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         "custom" : { "source" : screen}
       };
       this.mb.publish(OO.EVENTS.DISCOVERY_API.SEND_DISPLAY_EVENT, eventData);
-      if (screen == CONSTANTS.SCREEN.END_SCREEN) {
```

Comment:
I need a second opinion about this. I couldn't figure out why this is overridden instead of using the value defined on the skin.json, but this prevented the end screen's countdown timer from being disabled.